be/src/runtime/memory/lru_cache_policy.h

```diff
@@ -120,14 +120,17 @@ class LRUCachePolicy : public CachePolicy {
 
     uint64_t new_id() { return _cache->new_id(); };
 
+    // Subclass can override this method to determine whether to do the minor or full gc
```

Comment:
inaccurate comment. gc type only related to memory status, minor gc exectue `prune_stale` and full gc exectue`prune_all`.
`exceed_prune_limit` only used for whether to exectue this cache prune.

Reply:
We are talking about the same thing